gammapy/catalog/core.py

```diff
@@ -214,17 +214,28 @@ def _make_source_object(self, index):
         data = table_row_to_dict(self.table[index])
         data[SourceCatalogObject._row_index_key] = index
 
+        fp_energy_edges = getattr(self, "flux_points_energy_edges", None)
+
+        if fp_energy_edges:
+            data["fp_energy_edges"] = fp_energy_edges
+        
         hist_table = getattr(self, "hist_table", None)
         hist2_table = getattr(self, "hist2_table", None)
 
+
         if hist_table:
-            data["time_axis"] = TimeMapAxis.from_table(hist_table, format="fermi-fgl")
```

Comment:
Has the time info been removed from the latest data release?

Reply:
No but it may be safer to assume that it can be missing  
(for example the 1 year light-curve was add in rev 3 of DR1  see the table https://fermi.gsfc.nasa.gov/ssc/data/access/lat/8yr_catalog/)